lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Tady je otázka, jestli vlastně chceme vykládat tenhle a následující příkazy, když z pohledu datový analýzy asi nejsou moc zajímavý.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Jeden odstavec o těchto příkazech bych zmínil. Něco jako že SQL se používá i na tvorbu a úpravy databází samotných, ale že to my znát v analýze nepotřebujeme.</p>